dll/appcompat/apphelp/sdbapi.c

```diff
@@ -151,7 +151,7 @@ void WINAPI SdbpFlush(PDB pdb)
 
 DWORD SdbpStrlen(PCWSTR string)
 {
-    return wcslen(string);
+    return (DWORD)wcslen(string);
```

Comment:
@learn-more : is this an internal function? Any ideas about making it returning SIZE_T instead?

Reply:
It's internal, so I can just update it to be `size_t`

Reply:
In fact I have doubts that this would fix things. It would just move the cast from one location to a dozen others that use this function. Then you need to fix all of those. If you are a purist that thinks that 8GB C-strings are a legitimate thing that could happen, then you would need to do it, I guess.

Reply:
Either way is fine with me